linalg/kernels/vector.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifndef MFEM_VECTOR_KERNELS
+#define MFEM_VECTOR_KERNELS
+
+namespace mfem
+{
+
+void kVectorMapDof(const int, double*, const double*, const int*);
```

Comment:
Long names that begin with `kVector` are too C-like. What does the group think of putting these in namespaces, e.g. this could then be `mfem::kernels::vector::MapDof`.